### ISSUE TITLE
Extensions/Inkstitch not showing

### ISSUE THREAD

**Issue by Jsdk2** [inkstitch/inkstitch#1262]
I was watching video on inkstitch and the guy went to Extensions/inkstitch/params. I go to Extensions/ and there is no Inkstictch. 
What am I doing wrong?

Thanks so much!
Julie

**Comment by kaalleen** [comment]
How did you install Ink/Stitch and which system do you use?

**Comment by Jsdk2** [comment]
Thanks!
I downloaded it and opened it.  I am on Windows 10. I read about moving to some folder, but didn't understand.  : (  : (

Could that be the problem?

**Comment by Jsdk2** [comment]
My son was able to figure it out. Thanks!

**Comment by RosiRoo** [comment]
I downloaded the latest V2 of inkstitch on my iMac. So it seems to me that Inkscape doesn't like the Subfolder "Inkstitch" in the extension folder.  When I deleted the subfolder and just pasted the add-files and main Inkstitch file, it worked.  
Also - it didn't install my Thread Color palette automatically - had to copy the files directly into the palette extension.  I don't know what else didn't install correctly since I am just NEW to Inkscape and inkstitch!   But I got it to show up in the extension menu, and I can see the stitch animation in Parame. :).  A lot of learning to do. Thanks!
